### PR TITLE
dr-autosync: add configuration description

### PR DESCRIPTION
**Pull request by disksing**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

 add dr-auto-sync configuration description

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -242,6 +244,8 @@ cat default.json
 + `primary-replicas` 是主 AZ 上 Voter 副本的数量。
 + `dr-replicas` 是从 AZ 上 Voter 副本的数量。
 + `wait-store-timeout` 是当出现网络隔离或者故障时，切换到异步复制模式的等待时间。如果超过这个时间还没恢复，则自动切换到异步复制模式。默认时间为 60 秒。
++ `wait-recover-timeout` 是当网络恢复后，切换回 `sync_recover` 模式的等待时间。默认时间为 0 秒。
```
Comment (qiancai) [inline two-data-centers-in-one-city-deployment.md:247]:
```suggestion
+ `wait-recover-timeout` 是当网络恢复后，切换回 `sync-recover` 状态的等待时间。默认为 0 秒。
```

Comment (qiancai) [reply two-data-centers-in-one-city-deployment.md:247]: 参考：
image

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -242,6 +244,8 @@ cat default.json
...
++ `pause-region-split` 用于控制在 `async_wait` 和 `async` 状态是否暂停 Region 的 split 操作，暂停 Region split 可以防止在 `sync_recover` 状态同步数据时从属区域出现短暂的部分数据缺失。默认为 false。
```
Comment (qiancai) [inline two-data-centers-in-one-city-deployment.md:248]:
```suggestion
+ `pause-region-split` 用于控制在 `async_wait` 和 `async` 状态下是否暂停 Region 的 split 操作。暂停 Region split 可以防止在 `sync-recover` 状态同步数据时从属区域出现短暂的部分数据缺失。默认为 `false`。
```

**Review by qiancai — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

File: two-data-centers-in-one-city-deployment.md [hunk 3]
```diff
@@ -242,6 +243,7 @@ cat default.json
 + `primary-replicas` 是主 AZ 上 Voter 副本的数量。
 + `dr-replicas` 是从 AZ 上 Voter 副本的数量。
 + `wait-store-timeout` 是当出现网络隔离或者故障时，切换到异步复制模式的等待时间。如果超过这个时间还没恢复，则自动切换到异步复制模式。默认时间为 60 秒。
++ `pause-region-split` 用于控制在 `async_wait` 和 `async` 状态下是否暂停 Region 的 split 操作。暂停 Region split 可以防止在 `sync-recover` 状态同步数据时从属区域出现短暂的部分数据缺失。默认为 `false`。
```
Comment (qiancai) [inline two-data-centers-in-one-city-deployment.md:246]:
```suggestion
+ `pause-region-split` 用于控制在 `async_wait` 和 `async` 状态下是否暂停 Region 的 split 操作。暂停 Region split 可以防止在 `sync-recover` 状态同步数据时从属 AZ 出现短暂的部分数据缺失。默认为 `false`。
```

**Pull request merged**